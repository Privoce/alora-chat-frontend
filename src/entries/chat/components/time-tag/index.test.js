import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	TimeTagComponent
} from 'entries/chat/components';

describe('<TimeTagComponent />', () => {
	const modelI = {
		text: 'today'
	};

	const getWrapper = (model) => {
		return shallow((
			<div>
				<TimeTagComponent {...model} />
			</div>
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
