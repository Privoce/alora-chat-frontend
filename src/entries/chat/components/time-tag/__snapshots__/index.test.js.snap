// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeTagComponent /> renders correctly 1`] = `
<div>
  <TimeTagComponent
    text="today"
  />
</div>
`;
